back/app/controllers/web_api/v1/phases_controller.rb

```diff
@@ -143,6 +143,7 @@ def phase_params
         voting_term_plural_multiloc: CL2_SUPPORTED_LOCALES,
         native_survey_title_multiloc: CL2_SUPPORTED_LOCALES,
         native_survey_button_multiloc: CL2_SUPPORTED_LOCALES,
+        form_success_multiloc: CL2_SUPPORTED_LOCALES,
```

Comment:
Named it `form_success_multiloc` in case we want to use it for other participation methods in the future